### DISCUSSION TITLE
Selection key bind.

### DISCUSSION THREAD

**Discussion by semi710** [Aloxaf/fzf-tab#285] [Q&A] [answered]
I use `ctrl+j` `ctrl+k` to move between fzf but if I want to select any thing I have to press tab that makes me leave the ctrl key and is a bit annoying to me is it possible to map ctrl+tab key to do the selection ?

**Comment by Aloxaf** [comment, marked as answer]
Unfortunately, fzf doen't support such a key binding.

Here are all available key bindings: https://man.archlinux.org/man/fzf.1#AVAILABLE_KEYS:_(SYNONYMS)
